### ISSUE TITLE
Bug: Support Android Gradle Plugin 8+

### ISSUE THREAD

**Issue by orkun1675** [Pyozer/introduction_screen#209]
Can we update the introduction_screen dependency on `flutter_keyboard_visibility` to `6.0.0` or later?

This will allow us to upgrade AGP to `8.2.2`

**Comment by rekire** [comment]
This will become more relevant when people start updating the Flutter SDK to 6.19 and make the transition to the plugin blocks in gradle.

**Comment by ghenry** [comment]
Can someone try it?

**Comment by rekire** [comment]
I added this to my `pubspec.yaml` and it works fine for me:
```yaml
dependency_overrides:
  flutter_keyboard_visibility: ^6.0.0
```

However I do not suggest that you should use this for production.

**Comment by braniii** [comment]
I am not sure, but since the dependency `flutter_keyboard_visibility` was updated on the master branch, one workaround could be to use git for resolving this package:

```yaml

  introduction_screen:
    git:
      url: https://github.com/Pyozer/introduction_screen.git
      ref: master
```

**Comment by Pyozer** [comment]
Sorry for the delay..

Version 3.1.13 published with flutter_keyboard_visibility 6.0.0
